Resources/Audio/_Harmony/StationEvents/attributions.yml

```
@@ -2,3 +2,8 @@
   license: "CC-BY-SA-3.0"
   copyright: "Created by pigeonbeans"
   source: "https://soundcloud.com/pigeonbeans/moodswings"
+
+- files: ["blatantly_nuclear.ogg"]
+  license: "Blatantly Nuclear by mrjajkes is licensed under the following terms: free use and modification with proper credit to the original creator, excluding usage by FulpStation, White Dream, Wizard's Den, and any ERP/sexually-oriented servers, or for commercial use unless granted permission by the author. For commercial use, please contact the author to discuss details."
+  copyright: "Created by mrjajkes"
```

Otherwise `RGA schema validator` fails
```suggestion
  license: "Custom"
  copyright: "Blatantly Nuclear by mrjajkes is licensed under the following terms: free use and modification with proper credit to the original creator, excluding usage by FulpStation, White Dream, Wizard's Den, and any ERP/sexually-oriented servers, or for commercial use unless granted permission by the author. For commercial use, please contact the author to discuss details."
```